### ISSUE TITLE
Need a table to link user and user subscriptions

### ISSUE THREAD

**Issue by franck-boullier** [unee-t/frontend#280]
# Overview:
In order to manage user subscriptions, we need a way to record the subscription information for each user.

One (easy) way to do that is to create a maping table between user id and subscription id.

# The information we need:
- User id
- Subscription plan id (default is the subscription id for the Unee-T Standard plan)
- Subscription start date
- Subscription end date (default is NULL)
- Payment platform id: how was this subscription paid for. This is the id in the table which will list all our payment options.
- Payment confirmation id: the id of the transaction on the payment platform. This will allow us to find detailed transaction information if and when needed.

**Comment by nbiton** [comment]
We need to integrate a payment solution first

**Comment by franck-boullier** [comment]
>We need to integrate a payment solution first

@nbiton: no
happy to discuss this if need be

**Comment by nbiton** [comment]
Do you mean it should be done manually? (adding this information)

**Comment by kaihendry** [comment]
Can we get a Stripe setup first, it's possible to bill users manually without any integration at first which is really nice.

**Comment by franck-boullier** [comment]
> Can we get a Stripe setup first, it's possible to bill users manually without any integration at first which is really nice.

Absolutely: we already have a Stripe account and the plans we need are already configured there
